src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1164,20 +1042,13 @@ namespace cryptonote
     }
     // for version > 1, ringct signatures check verifies amounts match
 
-    if(!keeped_by_block && get_transaction_weight(tx) >= m_blockchain_storage.get_current_cumulative_block_weight_limit() - CRYPTONOTE_COINBASE_BLOB_RESERVED_SIZE)
```

Comment:
Yes... although it might be good to double check: there should be a strictly more stringent *consensus* tx weight test in `ver_non_input_consensus()` for all transactions in hard fork >= v8.